examples/window-app/efr32/src/WindowAppImpl.cpp

```diff
@@ -235,7 +193,26 @@ void WindowAppImpl::PostEvent(const WindowApp::Event & event)
 {
     if (mQueue)
     {
-        if (!xQueueSend(mQueue, &event, 1))
+        BaseType_t status;
+        if (xPortIsInsideInterrupt())
+        {
+            BaseType_t higherPrioTaskWoken = pdFALSE;
+            status                         = xQueueSendFromISR(mQueue, &event, &higherPrioTaskWoken);
```

Comment:
You have to call `fromIsR` freeRTOS functions when it is called from a ISR else it cause a freeRTOS assert. You Cannot lock in a ISR. Before it was doing something similar but the button press was starting a short timer for  "debouncing" and at the timer expirency, it would do the queueSend.

This debouncing is not needed in the app nor with the new driver. So we can do the queueSend right away